### PR TITLE
Add dynamic data

### PR DESCRIPTION
**Pull request by gharti2704**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
 .DS_Store
-node_modules
```
Comment (alexzinnde) [inline .gitignore:2]: We want to git to ignore the node_modules folder. Please add  nod_moduels to the .git_ignore

File: server/app.js [hunk 2]
```diff
@@ -38,4 +38,18 @@ app.get('/productOverview/:productId', (req, res) => {
     .catch((err) => res.status(500).send(err));
 });
 
+
+app.get('/:id', (req, res) => {
```
Comment (alexzinnde) [inline server/app.js:42]: I suggest you change your endpoint to be more specific to your widget. I suggest something like: 
`app.get('/relatedProducts/:id'`...

File: server/app.js [hunk 3]
```diff
@@ -38,4 +38,18 @@ app.get('/productOverview/:productId', (req, res) => {
...
+  const id = req.params.id;
+  api.getRelatedProductId(id, (err, response) => {
```
Comment (alexzinnde) [inline server/app.js:44]: If you can, I suggest refactoring your callback to a promise pattern, so our code has more uniformity.

File: src/components/related-items-comparison/ProductInformation.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const ProductInformation = ({product}) => {
+  console.log(product, 'Hi! You clicked me!!!!');
```
Comment (alexzinnde) [inline src/components/related-items-comparison/ProductInformation.jsx:4]: let's remove console.logs unless absolutely necessary

File: src/components/related-items-comparison/RelatedProductCard.jsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
+
+// require('dotenv').config();
```
Comment (alexzinnde) [inline src/components/related-items-comparison/RelatedProductCard.jsx:2]: You don't need to require dotenv inside your component. Only need this inside the server to hold the github token info

File: src/components/related-items-comparison/RelatedProductCard.jsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+import React from 'react';
+import ProductInformation from './ProductInformation.jsx';
+import axios from 'axios';
+
+
+class RelatedProductCard extends React.Component {
+  //const [product, setProduct] = useState([]);
```
Comment (alexzinnde) [inline src/components/related-items-comparison/RelatedProductCard.jsx:9]: I don't think you can use useState inside a class component.

**Review by alexzinnde — CHANGES_REQUESTED** [review]

File: src/components/related-items-comparison/RelatedProductCard.jsx (hunk 6 above)
Comment (gharti2704) [reply src/components/related-items-comparison/RelatedProductCard.jsx]: Error fixed.

File: src/components/related-items-comparison/Outfit.jsx [hunk 7]
```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
+
+
+const Outfit = ({ product }) => {
+  let products = [];
+  products = products.concat(product);
+  console.log(products);
```
Comment (alexzinnde) [inline src/components/related-items-comparison/Outfit.jsx:8]: Remove the console.log here.

File: src/components/related-items-comparison/RelatedProductCard.jsx [hunk 8]
```diff
@@ -1,66 +1,117 @@
...
-
+    this.prev = this.prev.bind(this);
+    this.next = this.next.bind(this);
+    this.getProduct = this.getProduct.bind(this);
   }
 
-
-
   clickHandler(id) {
+    console.log(id);
```
Comment (alexzinnde) [inline src/components/related-items-comparison/RelatedProductCard.jsx:25]: Remove the console.log here

**Review by alexzinnde — APPROVED** [review]

**Pull request merged**